src/csharp/Grpc.Core/Internal/AsyncCall.cs

```diff
@@ -212,7 +215,7 @@ public void StartDuplexStreamingCall(Metadata headers)
         }
 
         /// <summary>
-        /// Sends a streaming request. Only one pending send action is allowed at any given time.
+        /// Sends a streaming request. Only one pending send action is allologger any given time.
```

Comment:
Yes, something really weird was going on with refactor feature in Monodevelop IDE. Thanks for spotting this.